src/ontology/uberon-edit.obo

```diff
@@ -128290,6 +128290,7 @@ xref: NCIT:C33426
 xref: SCTID:361365004
 xref: UMLS:C0225696 {source="ncithesaurus:Pulmonary_Acinus"}
 xref: Wikipedia:Acinus#The_Lungs
+is_a: UBERON:0034922 ! cell cluster
```

Comment:
The pulmonary acinus contain alveolar ducts and respiratory bronchioles, I don't think it should be classified as a cell cluster. Should it also be an anatomical entity?

Reply:
happy to put it as anatomical entity :) only put cell cluster cause I searched up acinus and it said cell cluster.

Reply:
Yes, the pulmonary acinus is a bit different than other acini.